MentoringHomeWork/HW4/Task3/Sum3/Sum3.cs

```diff
@@ -0,0 +1,72 @@
+using System.Security;
+
+namespace Task3.Sum3;
+
+public class Sum3
+{
+    public void Task3()
+    {
+        var result = 0;
+        Getnumber();
+        CheckSum();
+        return;
+
+
+        void Getnumber()
+        {
+            Console.WriteLine("Please enter the first number: ");
+
+            if (int.TryParse(Console.ReadLine(), out var number))
+            {
+                Console.WriteLine("Please enter the second number: ");
+
+                if (int.TryParse(Console.ReadLine(), out var number2))
+                {
+                    result = number + number2;
+                }
+                else
+                {
+                    Console.WriteLine("Error, incorrect input for the second number. Please try again.");
+                    Task3();
+                }
+            }
+            else
+            {
+                Console.WriteLine("Error, incorrect input for the first number. Please try again.");
+                Task3();
+            }
+        }
+
+        void CheckSum()
```

Comment:
![image](https://github.com/user-attachments/assets/b2ffc0d2-03d1-48e6-9128-264861166ef4)
minor - not required change

Reply:
resolved